### PR TITLE
[TT-8298] Correct typos in import.md

### PR DESCRIPTION
**Pull request by andyo-tyk**
Replaced apiid and apiname with the correct fields (api_id and api_name); fixed a couple of typos

### CODE REVIEW

**Review by dcs3spp — APPROVED** [review]
The changes in the examples for import organisation look ok to me. For this PR before merging and release will wait until peer review has happened. I am not sure why organisation was capitalised? Once peer reviewed let me know and we can merge and release.

**Comment by andyo-tyk** [comment]
> The changes in the examples for import organisation look ok to me. For this PR before merging and release will wait until peer review has happened. I am not sure why organisation was capitalised? Once peer reviewed let me know and we can merge and release.

I capitalised Organisation, because here we are referring to a Tyk Organisation - rather than the user's organisation (company). The use of a "common word" for a Tyk concept isn't ideal, but it's what's been used for years so I would rather capitalise in an attempt to flag that it has a specific meaning.

**Comment by letzya** [comment]
seeing this PR reminded me that we also have a swagger with these typos https://tyk.io/docs/dashboard-admin-api/

**Comment by letzya** [comment]
> Organisation, because here we are referring to a Tyk Organisati

@andyo-tyk not sure I understand what you mean here. What is User Org and what is Tyk org? but happy to merge

**Comment by letzya** [comment]
Also - why not merge new? @andyo-tyk

**Comment by andyo-tyk** [comment]
Tyk Organisation - https://tyk.io/docs/basic-config-and-security/security/dashboard/organisations/#concept-dashboard-organisations
The word "organisation" is commonly used to refer to a company, so when I said "the user's organisation (company)" that's what I meant - https://dictionary.cambridge.org/dictionary/english/organization?q=organisation

**Review by jeffy-mathew — APPROVED** [review]

File: tyk-docs/content/tyk-apis/tyk-dashboard-admin-api/import.md [hunk 1]
```diff
@@ -19,12 +19,12 @@ To enable this feature, the minimum required versions for the Gateway and Dashbo
...
 **Warning**  
 
-In a production environment, you will need to change the default `admin_Secret` value that is called by the `admin-auth` header in your `tyk_analytics.conf` file. This is located in `/opt/tyk-dashboard`.
+In a production environment, you will need to change the default `admin_secret` value that is called by the `admin-auth` header in your `tyk_analytics.conf` file. This is located in `/opt/tyk-dashboard`.
 {{< /warning >}}
 
 #### Import Organisation
 
-The organisation object is the most fundamental object in a Tyk setup, all other ownership properties hang off the relationship between an organisation and it's APIs, Policies and API Tokens.
+The Organisation object is the most fundamental object in a Tyk setup, all other ownership properties hang off the relationship between an Organisation and its APIs, Policies and API Tokens.
```
Comment (letzya) [inline tyk-docs/content/tyk-apis/tyk-dashboard-admin-api/import.md]:
```suggestion
The [Organisation object]({{<ref "basic-config-and-security/security/dashboard/organisations#concept-dashboard-organisations" >}}) is the most fundamental object in a Tyk setup, all other ownership properties hang off the relationship between an Organisation and its APIs, Policies and API Tokens.
```

**Review by letzya — CHANGES_REQUESTED** [review]
Please add the admin API swagger changes as well to avoid confusion for the users

File: tyk-docs/content/tyk-apis/tyk-dashboard-admin-api/import.md (hunk 1 above)
Comment (andyo-tyk) [reply tyk-docs/content/tyk-apis/tyk-dashboard-admin-api/import.md]: Happy to accept this - but note this was just something I noticed when modifying the page elsewhere. This wasn't a full review/rewrite of the page hence I didn't go looking for all the possible improvements on this page.

**Comment by andyo-tyk** [comment]
> Please add the admin API swagger changes as well to avoid confusion for the users

?

File: tyk-docs/content/tyk-apis/tyk-dashboard-admin-api/import.md (hunk 1 above)
Comment (letzya) [reply tyk-docs/content/tyk-apis/tyk-dashboard-admin-api/import.md]: 100%

**Comment by letzya** [comment]
@andyo-tyk @jeffy-mathew  I added it, please review.

**Comment by dcs3spp** [comment]
@andyo-tyk @jeffy-mathew Have the changes added by @letzya being reviewed? If ok I will then merge and release

**Comment by andyo-tyk** [comment]
As far as I know, yes - need @letzya to confirm if there's anything outstanding.

**Comment by letzya** [comment]
approved outside this chat

**Pull request merged**